examples/kubernetes/templates/v1/cilium-microk8s-ds.yaml.sed

```diff
@@ -0,0 +1,244 @@
+          readOnly: true
+          # Needed by ip[6]tables when loading kernel modules
+        - mountPath: /sbin/modprobe
+          name: sbin-modprobe
+          readOnly: true
+      dnsPolicy: ClusterFirstWithHostNet
+      hostNetwork: true
+      hostPID: false
+      initContainers:
+      - command:
+        - /init-container.sh
+        env:
+        - name: CILIUM_ALL_STATE
+          valueFrom:
+            configMapKeyRef:
+              key: clean-cilium-state
+              name: cilium-config
+              optional: true
+        - name: CILIUM_BPF_STATE
+          valueFrom:
+            configMapKeyRef:
+              key: clean-cilium-bpf-state
+              name: cilium-config
+              optional: true
+        image: docker.io/cilium/cilium-init:__CILIUM_INIT_VERSION__
+        imagePullPolicy: IfNotPresent
+        name: clean-cilium-state
+        securityContext:
+          capabilities:
+            add:
+            - NET_ADMIN
+          privileged: true
+        volumeMounts:
+        - mountPath: /sys/fs/bpf
+          name: bpf-maps
+        - mountPath: /var/run/cilium
+          name: cilium-run
+      restartPolicy: Always
+      serviceAccount: cilium
+      serviceAccountName: cilium
+      terminationGracePeriodSeconds: 1
+      tolerations:
+      - operator: Exists
+      volumes:
+        # To keep state between restarts / upgrades
+      - hostPath:
+          path: /var/run/cilium
+          type: DirectoryOrCreate
+        name: cilium-run
+        # To keep state between restarts / upgrades for bpf maps
+      - hostPath:
+          path: /sys/fs/bpf
+          type: DirectoryOrCreate
+        name: bpf-maps
+        # To read labels from containerD containers running in the host
+      - hostPath:
+          path: /var/snap/microk8s/common/run/containerd.sock
+          type: Socket
+        name: containerd-socket
+        # To install cilium cni plugin in the host
+      - hostPath:
+          path: /opt/cni/bin/
+          type: Directory
+        name: cni-path
+        # To install cilium cni configuration in the host
+      - hostPath:
+          path: /var/snap/microk8s/current/args/cni-network/
+          type: Directory
+        name: etc-cni-netd
+        # To be able to load kernel modules
+      - hostPath:
+          path: /lib/modules
+          type: Directory
+        name: lib-modules
+        # To be able to load ip[6]tables kernel modules
+      - hostPath:
+          path: /sbin/modprobe
+          type: File
+        name: sbin-modprobe
+        # To read the etcd config stored in config maps
+      - configMap:
+          defaultMode: 420
+          items:
+          - key: etcd-config
+            path: etcd.config
+          name: cilium-config
+        name: etcd-config-path
+        # To read the k8s etcd secrets in case the user might want to use TLS
+      - name: etcd-secrets
+        hostPath:
+          path: /var/snap/microk8s/current/certs/
```

Comment:
AFAIK you are deploying the etcd standalone, by doing it so we don't need to mount this directory.

Reply:
This will be dropped by reusing the containerd yaml, see latest updates. Thanks for the review!